Python/optimizer_bytecodes.c

```diff
@@ -651,6 +651,28 @@ dummy_func(void) {
         }
     }
 
+    op(_RETURN_GENERATOR, ( -- res)) {
+        SYNC_SP();
+        ctx->frame->stack_pointer = stack_pointer;
+        frame_pop(ctx);
+        stack_pointer = ctx->frame->stack_pointer;
+        OUT_OF_SPACE_IF_NULL(res = sym_new_unknown(ctx));
```

Comment:
Am I correct that this is the only line that differs from `_POP_FRAME`? The duplication of so much code is unfortunate. At the same time I don't see a better solution. Maybe add a comment to both explaining they need to be kept in sync?

Reply:
A good candidate for refactoring, but I'd like to get #118095 done first so that we more easily see the common patterns.